mantidproject/mantid
# Added event sliced mergeing support

**Don't Merge yet want to tidy commits**
The SANS reduction framework was failing when you attempted to do a merged reduction of event sliced data. 

**To test:**
- Get the SANS2D dataset from \\olympic\Babylon5\Scratch\Matthew Andrew\GUI_TEST
- Add the folder to the directories path in mantid.

- Open the old SANS GUI, Interfaces-> SANS
- Select SANS2DTUBES as the instrument
- Select Batch Mode
- Click on load user file navigate to GUI_TEST and load USER_SANS2D_154E_2p4_4m_M3_Xpress_8mm_SampleChanger_REAR.txt

- Click on load batch file, load batch_file

- In the time slices box enter 1:1:3
- In the reduction settings tab selected merged as the reduction mode (labelled Detector Bank on the old gui).
- Click 1D Reduce
- Confirm that workspace groups called line_1_reduction_front, line_1_reduction_rear and line_1_reduction_merged are produced.

- Open the new sans Gui and repeat the above reduction but change the output name to 'New' and select compatibility mode from the settings->General. Scale, Event Slice, Sample tab.

- Confirm that a workspace group called New_rear is produced (Note this naming seems to be a bug with the new GUI)

- Confirm that the individual workspaces within New_rear and line_1_reduction_merged are the same.



Fixes #19196 . 

**Release Notes** 
Updated release notes.


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
